src/index.html

```diff
@@ -1,13 +1,376 @@
+                </h3>
+                <div class="testimonial__role">Chief Design Officer</div>
+              </div>
+              <p class="testimonial__about">
+                When my lovely valley of steam rises and half-day sun is above an impermeable
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55" class="contact-us__info-link"
+                >
+                +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com" class="contact-us__info-link"
+                >
+                hello@miami.com
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/YeRn2tmYW1iZcxhp9" class="contact-us__info-link"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+            <div class="grid__item grid__item--8-12">
+              <form action="">
+                <input
+                  name="name"
+                  placeholder="Name"
+                  type="text"
+                  class="form-field contact-us__field"
+                >
+                <input
+                  name="email"
+                  placeholder="Email"
+                  type="email"
+                  class="form-field contact-us__field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contact-us__field contact-us__field--last form-field form-field--textarea"
+               ></textarea>
+                <button class="button button--primary">
+                  Consultation
+                </button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <a href="/" class="logo">
+        <img
```

Comment:
fix it
![image](https://user-images.githubusercontent.com/57729155/134727377-83a38fb0-2398-49f6-b7bd-6314ce07c86a.png)